geps/gep-1897.md

```diff
@@ -125,7 +132,251 @@ This GEP is the outcome of the TLS use cases #4 and #5 in
+[GEP-713](https://gateway-api.sigs.k8s.io/geps/gep-713/), that the implementation
+should watch the connections to a specified TargetRef (Service), and if the Service matches a TLSConnectionPolicy, then
+assume the connection is TLS, and verify that the TargetRef’s certificate can
+be validated by the provided trusted CA certificates before the connection is made.  On the question of how to signal
+that there was a failure in the certificate validation, this is left up to the implementation to return a response error
+that is appropriate, such as one of the HTTP error codes: 400 (Bad Request), 401 (Unauthorized), 403 (Forbidden), or
+other signal that makes the failure sufficiently clear to the requester without revealing too much about the transaction,
+based on established security requirements.
+
+All policy resources must include TargetRef with the fields specified 
+[here](https://github.com/kubernetes-sigs/gateway-api/blob/a33a934af9ec6997b34fd9b00d2ecd13d143e48b/apis/v1alpha2/policy_types.go#L24-L41).
+In an upcoming extension to TargetRef, policy resources _may_ also choose to include `SectionName` and/or `Port` in the
+TargetRef following the same mechanics as `ParentRef`.
+
+Not covered here, but possible to add would be additional configuration options mentioned in [SIG-NET Gateway API: TLS
+to the K8s.Service/Backend](https://docs.google.com/document/d/1RTYh2brg_vLX9o3pTcrWxtZSsf8Y5NQvIG52lpFcZlo).  All of
+these are currently implementation-dependent, with the following recommended defaults:
+
+- Server Name Indication: allow passing of the server name through server name indication, in the TLS transaction, to
+assist with selection of certificates when several hosts share the same IP address. (default to enabled)
+- Subject Alternative Name certificates: allow authentication with a single certificate that can serve multiple domain
+names that are intended to represent the same endpoint.  For example, these five SAN hosts: example.com, 
+www.example.com, example.net, IPv4 x.x.x.x, and IPv6 x.x.x.x.x.x.x.x all resolve to the same endpoint.
+(default to enabled)
+- Version: specifies the minimum TLS version that the connection may use (default TLSv1.2)
+- Ciphers: specifies enabled ciphers to be used in TLS exchanges. (default to align with default TLS version chosen for
+Version, e.g. ciphers mentioned for TLS version 1.1, 1.2, or 1.3 as described in 
+[RFC4346 TLS 1.1](https://datatracker.ietf.org/doc/html/rfc4346#appendix-A.5),
+[RFC 5246 TLS 1.2](https://datatracker.ietf.org/doc/html/rfc5246#appendix-A.5), or
+[RFC 8446 TLS 1.3](https://datatracker.ietf.org/doc/html/rfc8446#appendix-B.4).
+- Visibility: the visibility level could be `all` or `none`, and represents that if the connection failed, 
+the implmentation would drop the connection silently if the visibility level were `none`, or report an error if the
+visibility level were `all`.  (default to `all`)
```

Comment:
I don't think this content is needed in the GEP, and they seem like they could be more controversial than they're worth. I'd rather leave them out until we actually provide a way to configure any of these settings.

Reply:
I will be keeping and expanding the discussion on AllowedSANs and TLSVersions, which have been added.